templates/environment/versions/cf-v1.3.0.yml

```diff
@@ -0,0 +1,407 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+Metadata:
+  Version: 'v1.3.0'
+
```

Comment:
Would you mind getting rid of these newlines between params/resources so that we can reduce the size as much as we can until we upload the env template as well to s3 🙏